.flake8

```diff
@@ -1,6 +1,6 @@
 [flake8]
-ignore = E203, E266
-max-line-length = 79
+ignore = E203, E266, W503
+max-line-length = 90
```

Comment:
why did you do this? remove all changes from this file

Reply:
But how do I move the line? Flake8 doesn't like that I'm wrapping a line on the addition operator